doc/adr/0003-directory-structure.md

```diff
@@ -0,0 +1,39 @@
+# 3. Define directory structure
+
+Date: 2020-12-01
+
+## Status
+
+Accepted
+
+## Context
+
+There will be many services contributed by multiple people. We need a common way of structuring the services
+so it's easy for the user to pick up any one service.
+
```

Comment:
Could you provide a context for user extensions ?

Reply:
added, please rereview